File: Backend/TopSys.TopConWeb.Domain/Entities/ResistenciaTipo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TopSys.TopConWeb.Domain.Enums;

namespace TopSys.TopConWeb.Domain.Entities
{
    public class ResistenciaTipo
    {
        public int Codigo { get; set; }

        public string Descricao { get; set; }

        public string  Abreviatura { get; set; }

        public EResistenciaVinculoTipo Vinculo { get; set; }
    }
}

File: Frontend/TopConWeb-AngularV8/src/app/classes/traco/resistencia-tipo.ts
export enum ETipoVinculoMpaConsumo {
  SEM_VINCULO = 0,
  MPA = 1,
  CONSUMO = 2,
}

export class ResistenciaTipo {
    codigo: number = 0;
    descricao: string = '';
    abreviatura: string = '';
    vinculo: ETipoVinculoMpaConsumo = ETipoVinculoMpaConsumo.MPA;
}